docs/sources/next/using-k6/test-authoring/create-tests-from-recordings/using-the-devtools-recorder.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Using the DevTools recorder'
+description: 'The DevTools recorder allows exporting k6 browser scripts from the recorder panel in Chrome DevTools.'
+weight: 01
+---
+
+The DevTools recorder lets you record user journeys using Chrome DevTools and then export them as a k6 script.
+
+## Before you start
+
+Before you start, consider the following:
+
+- [Be sure to record realistically](/test-authoring/create-tests-from-recordings/#be-sure-to-record-realistically)
+- [A hybrid approach for load testing websites](/test-authoring/create-tests-from-recordings/#consider-hybrid-approach-for-load-testing-websites)
+
+{{< admonition type="note" >}}
+
+Note that the DevTools recorder **does not require a cloud account**.
+
+{{< /admonition >}}
+
+## How to record
+
+1. Install the [Chrome](https://chrome.google.com/webstore/detail/grafana-k6-browser-record/fbanjfonbcedhifbgikmjelkkckhhidl) extension.
```

Comment:
```suggestion
1. Install the [Grafana k6 Browser Recorder Chrome extension](https://chrome.google.com/webstore/detail/grafana-k6-browser-record/fbanjfonbcedhifbgikmjelkkckhhidl).
```

Reply:
Also, do we need to update this link to the new extension once it's published?

Reply:
> Also, do we need to update this link to the new extension once it's published?

Yes. I will add a commit to this PR once I know the final URL.